docs: restructure docs

matifali opened the pull request:
Closes #13434 
Supersedes #14182

ethanndickson committed:
fix: set network telemetry client version on server (#14376)

matifali committed:
rebase

matifali committed:
fmt

matifali committed:
make gen

matifali committed:
update tutorials paths

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
gen

matifali committed:
fix a typo check

matifali committed:
add automation page and fiox links in workspace page

matifali committed:
fmt & gen

matifali committed:
fix misc links

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
Merge branch 'main' into restructure-new

matifali committed:
spellings

matifali committed:
clean up and fix some links

ethanndickson committed:
port #14548

ethanndickson committed:
link troubleshooting

matifali committed:
add TODOs

matifali committed:
more TODO

bpmct committed:
Update manifest.json to match formatting

- Improve JSON formatting by adding line breaks for array items
- Shorten "Control Plane Configuration" to "Control Plane" for clarity

bpmct committed:
Update administration documentation

Remove outdated introduction and table of contents from the
administration README to improve clarity in the doc structure.

- Eliminated prerequisites and content list placeholders
- Redirected readers to the respective sections for detailed content

matifali committed:
link control plane configuraion in install docs

bpmct committed:
Update links to git provider configuration

Users require these links to correctly configure their git providers.

- GitHub Enterprise: ../configure.md#github-enterprise
- GitLab: ../configure.md#gitlab-self-managed
- BitBucket: ../configure.md#bitbucket-server
- Other Providers: ../configure.md#external-authentication

bpmct committed:
Rename "Workspaces" to "User Guides"

- Enhance document structure by renaming the main heading.
- Implement XML-style child linking for future content inclusion.

bpmct committed:
Fix broken link to provisioner authentication in docs

Correct the reference to provisioner authentication to ensure
users can locate the relevant documentation.

- Update the link from `#TODO` to `../provisioners.md`.

matifali committed:
add docker in workspaces

matifali committed:
port #14567

matifali committed:
move workspace-tags

stirby committed:
chore: update installation landing page (#14574)

Co-authored-by: Muhammad Atif Ali <atif@coder.com>

matifali committed:
`make fmt`

matifali committed:
Add icons to various documentation sections

Enhance navigation by adding relevant icons to improve visual
distinction and accessibility:

- Added `icon_path` for CLI, Docker, Kubernetes, OpenShift
- Enhanced cloud and other install methods with icons
- Updated user guides and control plane sections
- Added icons for infrastructure, users, and templates
- Included icons for provisioners, integrations, workspaces
- Added `icon_path` for networking, monitoring, security
- Included icons for automation and notifications sections

matifali committed:
add icons

matifali committed:
Update icon paths in manifest.json

This change updates the icon paths for various documentation sections
to better represent their content.

- Control Plane: ./images/icons/toggle_on.svg
- Monitoring: ./images/icons/speed.svg
- Automation: ./images/icons/plug.svg
- Notifications: ./images/icons/info.svg

matifali committed:
Add terms to VSCode settings for spell checking

- Ensure consistent spell-checking in the codebase
- Added the following terms:
  - `autoupdate`
  - `dbcrypt`
  - `dotfiles`
  - `filebrowser`
  - `jetbrains`
  - `knowledgebase`
  - `netcheck`
  - `portsharing`
  - `speedtest`
  - `unauthenticate`
  - `untar`
  - `userspace`
  - `workspaceproxies`

matifali committed:
Refactor external authentication docs

- Consolidate documentation for external authentication providers.
- Create a dedicated `external-auth.md` for detailed configuration steps.
- Update links in `configure.md` to reflect the changes for ease of navigation.

Additionally:

- Correct broken links.
- Enhance document structure for clarity and ease of use.

stirby committed:
Move notifications to monitoring

* Relocate the notifications page under the monitoring section.
* Update relevant links and image paths.

Changes:
- Rename notifications.md to monitoring/notifications.md.
- Adjust CLI reference links and image paths.
- Update documentation manifest to reflect the new path.

stirby committed:
fixed notifications link to feature stages

matifali committed:
Fix duplicate "About" entry in docs manifest

The manifest file erroneously included a duplicate "About" entry. This
commit removes the redundant entry to maintain documentation consistency.

matifali committed:
Refactor docs to improve initial user engagement

- Change "About" section to "Getting Started" for clarity.
- Update path to improve navigation for new users.

matifali committed:
Rename "Control Plane" to "Setup" in docs

- Improves clarity by using a more intuitive title
- "Setup" better reflects the content's purpose

matifali committed:
Refactor automation documentation

Relocate automation documentation to the reference section to improve
the organization of the docs. This makes the reference section the
go-to place for automation-related capabilities in Coder.

Changes:
- Moved and deleted `automation.md` from the admin section.
- Updated `manifest.json` to remove the automation entry.
- Transferred content from `automation.md` to `reference/README.md`.

This improves documentation accessibility and organizational clarity.

matifali committed:
Rename `configure.md` to `setup.md`

Reflect the name change in the manifest to improve clarity on user access configuration.

- Rename `docs/admin/configure.md` to `docs/admin/setup.md`
- Update manifest's path to reference the new file name

matifali committed:
Update link to external authentication doc

- Corrected a broken link in the setup documentation.
- Added external authentication entry to docs manifest.

This ensures consistency in navigation and accessibility.

stirby committed:
Add metrics documentation for Prometheus

* Provide detailed steps to set up metrics monitoring.
* Include links to Prometheus integration sections.
* Give an overview of default metrics available.

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

matifali committed:
Add icons for Access Workspaces and Scheduling sections

Enhance navigation by incorporating relevant icons for better visual distinction and accessibility.

- Access Workspaces: Added "access.svg"
- Workspace Scheduling: Added "stopwatch.svg"

These changes improve the user experience in navigating the documentation.

stirby committed:
docs: add introduction to user guides

Update `README.md` in the user guides section to include an introduction:
- Information on workspace access and personalization
- Clarification on what it does not include for better user direction

matifali committed:
Fix formatting issues in docs

- Corrected indentation in `external-auth.md`
- Improved line wrapping in `metrics.md`, `setup.md`, and `README.md`
- Fixed JSON formatting in `manifest.json`
- Enhanced script indentation in `reference/README.md`
- Removed unnecessary blank lines in `user-guides/README.md`

stirby committed:
make fmt

stirby committed:
merge

stirby committed:
Update admin README with setup and references

- Provide links for new users setting up control plane
- Offer reference guides for template creation, CLI, and API
- Point to FAQs and Tutorials for additional information

matifali committed:
Update workspace lifecycle docs with new links

matifali committed:
Add puzzle icon for integrations section

matifali committed:
chore: rename docs title from About to Home

matifali committed:
refactor: clean up and restructure docs links

stirby committed:
chore(docs): add security advisories section

Provide essential information for reporting vulnerabilities and list known security advisories with details on fixes and affected versions.

- Added instructions for reporting vulnerabilities
- Included a table to list known vulnerabilities with severity, fix version, and affected versions
- Added initial entry for API tokens invalidation issue

matifali committed:
Update Terraform code blocks to use `tf` language

stirby committed:
fix: update SCIM API key environment variable name

Update the environment variable for SCIM API key to `CODER_SCIM_AUTH_HEADER`
to ensure consistency with the latest configuration changes.

- Replace `CODER_SCIM_API_KEY` with `CODER_SCIM_AUTH_HEADER`

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
docs: fix line wrapping for dotfiles module section

- Improve readability by fixing line wrapping for the dotfiles module section.

stirby committed:
typos hotfix

EdwardAngert committed:
Merge remote-tracking branch 'origin' into rebase-restructure-new

EdwardAngert committed:
chore(docs): update JetBrains gateway setup (#14807)

- updates the JetBrains Gateway set up steps
- adjusts the relative image urls to the new directory structure

---------

Co-authored-by: EdwardAngert <17991901+EdwardAngert@users.noreply.github.com>

stirby committed:
chore(docs): removes dead navigation options (#14864)

Reverts manifest additions here

https://github.com/coder/coder/commit/d7ecab57ffbde9d07b50498c8314d4adeee4c59f#diff-0cbcfff19a4479219e722863d8f59891658d042d3fac32fe9d9a75d650ce0524

---------

Co-authored-by: EdwardAngert <17991901+EdwardAngert@users.noreply.github.com>

EdwardAngert committed:
Merge branch 'main' into restructure-new

bpmct committed:
Fix broken links in documentation

EdwardAngert committed:
fix image links in subdirs

EdwardAngert committed:
fix images

EdwardAngert committed:
fix images

EdwardAngert committed:
fix broken links

EdwardAngert committed:
update links

EdwardAngert committed:
update links

matifali committed:
Update REST API use cases link in postprocess script

matifali committed:
Fix broken API use cases link in README

matifali committed:
Update JFrog Artifactory offline deployment link

matifali committed:
fix dead links in jfrog-xray.md

matifali committed:
Fix template links in integration guide

matifali committed:
Update link for CI/CD pipeline management

matifali committed:
Add Git cloning guide to documentation manifest

matifali committed:
Update links in Git cloning tutorial for consistency

matifali committed:
Fix broken links and improve formatting in docs

Ensure documentation links are current and adjust line spacing for better readability across multiple integration and tutorial pages.

matifali committed:
Fix broken links in template tutorial docs

matifali committed:
Fix line wrapping in template tutorial doc

Improve line wrapping for better readability and alignment with style standards in the `template-from-scratch.md` tutorial document.

matifali committed:
Fix broken links and spelling in FAQs doc

matifali committed:
Fix IDE link in workspaces documentation

matifali committed:
Fix typo in workspace access README link

matifali committed:
Fix broken links in getting started guide

matifali committed:
Update Coder install link in first template guide

matifali committed:
Update remote desktops doc to enhance clarity

Enhance the remote desktops documentation by refining links and formatting for improved accessibility and understanding.

matifali committed:
Fix image path in remote desktops guide

matifali committed:
Fix SSH link in web IDEs documentation

matifali committed:
Update web IDE docs with new image paths

Refactor the web IDE documentation to include updated image paths that reflect the new directory structure.

matifali committed:
Update web IDE documentation for consistency

matifali committed:
Improve networking and SSH link paths in docs

matifali committed:
Remove unnecessary next steps section link

matifali committed:
Remove trailing newline from workspace access README

matifali committed:
Fix relative links in port forwarding guide

Updated relative paths to setup and SSH configuration links for clarity.

matifali committed:
Update configuration links in install docs

Refactor links in installation documents to point to the `setup.md` for improved clarity and consistency in user guidance.

matifali committed:
Fix configuration link paths in install docs

matifali committed:
Update validated architecture doc links

matifali committed:
fix(docs): standardize spelling for consistency

Ensure consistent spelling across documentation by updating "behaviour" to "behavior" to align with US English usage.

matifali committed:
Update links and icons in validated architectures docs

- Correct links for workspace and template update sections.
- Use ISO-8859-1 encoding for puzzle icon SVG.
- Rename "External Authentication" to "External Auth" in manifest.

EdwardAngert committed:
adjust links

EdwardAngert committed:
adjust links

matifali commented on docs/tutorials/faqs.md:
```diff
@@ -405,7 +405,7 @@ colima start --arch x86_64  --cpu 4 --memory 8 --disk 10
 
 Colima will show the path to the docker socket so we have a
-[community template](https://github.com/sharkymark/v2-templates/tree/main/docker-code-server)
+[community template](https://github.com/sharkymark/v2-templates/tree/main/src/docker-code-server)
```
We plan to move these to coder/templates once we make it public.

EdwardAngert committed:
test sentence case for next steps

EdwardAngert committed:
change-management next steps

EdwardAngert committed:
modules next steps

stirby committed:
added contributing, moved security warning to main contributing page

stirby committed:
added upgrading doc under install dir

stirby committed:
workspace proxies added under networking

stirby committed:
added quotas

stirby committed:
added HA mode

stirby committed:
added appearance as a subsection of setup

stirby committed:
removed legacy screenshot

stirby committed:
added telemetry to admin/setup, fixed up setup README

stirby committed:
removed unused notifications icon in monitoring

stirby committed:
changed references of admin/upgrade to install/upgrade

stirby committed:
moved git cloning guide out of folder

stirby committed:
removed mention of experiment in notifications

stirby committed:
fixed notifications structure and experiment references

stirby committed:
copied organizations from main

stirby committed:
copied images from organizations folder in main to images/admins/users/organizations

stirby committed:
removed empty groups heading

stirby committed:
fixed template permissions

stirby committed:
added template schedule settings, very likely broke some things

matifali committed:
Remove duplicate dotfiles.md file

This file has been deleted as it was redundant. Ensuring that no duplicate documentation exists helps maintain clarity and organization in the codebase.

matifali committed:
Add section for database encryption in docs manifest

matifali committed:
Fix relative links in Git cloning tutorial

matifali committed:
Remove placeholder tutorial READMEs

matifali committed:
Fix user schedule image path in workspaces.md

matifali committed:
Update doc links and add Jupyter module details

- Updated link formatting and paths in:
  - docs/admin/templates/extending-templates/modules.md
  - docs/tutorials/cloning-git-repositories.md
  - docs/user-guides/workspace-access/web-ides.md
- Added Jupyter Notebook setup details in web-ides.md
- Changed icon path for Airflow in web-ides.md

matifali committed:
Fix broken links in web IDEs documentation

Adjust the paths in the web IDEs section of user guides to reflect
recent directory changes.

matifali committed:
Move reverse-proxy docs from examples to docs

- Relocate Apache, Caddy, and NGINX configuration from examples.
- Update internal links for configuration guides to reflect new paths.
- Improve documentation structure by centralizing setup resources.

matifali committed:
Update cloud guide links in install README

matifali committed:
Refactor provisioners documentation

Refactor the provisioners documentation to improve clarity and guidance on setting up external provisioners. This includes:

- Emphasizing the benefits of using external provisioners over built-in ones.
- Streamlining instructions for setting up provisioners with scoped keys.
- Encouraging use of organization-scoped or user-scoped provisioners.
- Adding examples and instructions for different setup environments.
- Organizing content for better navigation and understanding.

matifali committed:
Add provider authentication section to docs manifest

EdwardAngert committed:
Merge branch 'main' into restructure-new

matifali commented on docs/admin/rbac.md:
@stirby this got added in re`base on `main.`` Do we want it directly under admin?

bpmct replied:
No, this should be in "template permissions" now.

bpmct replied:
https://github.com/coder/coder/blob/main/docs/templates/permissions.md

matifali commented on docs/admin/groups.md:
@stirby this got added in re`base on `main.`` Do we want it directly under admin?
This should go into admin/users/groups what do you think?

bpmct replied:
Yep. There is some new content that must be combined with https://github.com/coder/coder/blob/restructure-new/docs/admin/users/groups.md

bpmct replied:
woah, why is it empty? ^

matifali replied:
no idea just noticed.

matifali replied:
Added it.

matifali commented on docs/admin/users/oidc-auth.md:
This page also got heavily modified. In restructuring we want the newly added section in admin/users/RAEDME.md.

bpmct replied:
Yep!

matifali commented on docs/templates/process-logging.md:
We do not have templates in restructure as a top-level heading. Everything for templates should go to `admin/templates`

This specific page should go to `admin/security` or `admin/templates/extensing-templates`

bpmct replied:
👍🏼 extending templates

matifali commented on docs/licensing.md:
This should also go somewhere under `admin/licesning`

matifali commented on docs/manifest.json:
Manifest changes should be carefully reviewed too.

matifali committed:
Revert "Merge branch 'main' into restructure-new"

This reverts commit c7052d6766cb601d01519756e8239ae48fdceef5, reversing
changes made to 995d509bda42e1a90bf191127d29a8ecd3ce7566.

matifali committed:
docs: rebase `restructure-new` on `main` (#14960)

matifali committed:
Update external auth section for premium feature

matifali committed:
Add licensing and groups documentation

- Introduce a Licensing section to detail feature access with licenses.
- Document the Groups feature for user access management in templates.
- Update the manifest to include the new documentation sections.

matifali committed:
Add process logging documentation for workspaces

Include instructions for enabling and configuring workspace process logging on Kubernetes to help users monitor system-level processes.

matifali committed:
Update VSCode settings and improve documentation

- Added additional terms to ensure accurate spell-checking.
- Updated regex examples for clarity in documentation.
- Adjusted FAQs for correct terminology and consistency.

stirby committed:
fixup todos

matifali committed:
Remove groups documentation files

stirby committed:
Fix starter templates image path in docs

Correct the relative path for the starter templates image to ensure it loads properly in the managing templates documentation:

- Updated path from `../../images/start/starter-templates.png` to `../../../images/start/starter-templates.png`.

stirby committed:
Update workspace update policies documentation

Improve clarity and navigation within the workspace update policies section by incorporating the following changes:

- Removed outdated screenshot placeholder.
- Linked to the managing templates section to provide guidance on enforcing update policies.
- Clarified the process of updating workspaces when template admins require the latest version.

stirby committed:
remove outdated todo in install/docker

stirby committed:
workspace management

matifali committed:
Add icon for workspace management in manifest.json

stirby committed:
moved all content from admin/workspaces to user guides

stirby committed:
Merge branch 'main' into restructure-new

stirby committed:
manifest typo

matifali committed:
Remove old workspaces documentation

- The `workspaces.md` file was removed as it contained outdated
  information, and its content is now integrated into user guides.
- This improves documentation consistency and reduces redundancy.

matifali committed:
Update documentation links and remove extra lines

Refactor documentation links to ensure accuracy and remove extraneous lines for improved readability.

matifali committed:
Update architecture doc links for consistency

Ensure all internal documentation links in the architecture guide are consistent and accurate by adopting the relative path format used throughout the documentation. This enhances user navigation and reduces link errors.

matifali committed:
Fix scale test utility link path

matifali committed:
Update Markdown link paths for consistency

- Ensure consistent link formatting throughout documentation.
- Improve navigation by adopting a unified link structure.

matifali committed:
Refactor group and role sync documentation

- Split group and role sync details into a dedicated `group-sync.md`.
- Update links to reflect new file structure.
- Simplify OIDC configuration guidance for easier navigation.

stirby committed:
removed outdated user-guides files

matifali committed:
Update manifest for Group Sync documentation

matifali committed:
Rename `README.md` to `index.md` for documentation files

Modernize the documentation structure by renaming all `README.md` files to `index.md`. This provides a more consistent and intuitive navigation, particularly for web-based documentation systems where `index.md` is commonly used as the default entry point.

matifali committed:
Add trash icon and uninstall section to docs

Include a trash icon for the uninstall section in the documentation manifest to improve visual navigation for users.

matifali committed:
Update networking doc with connection guidelines

- Provide minimum requirements for clients to connect.

stirby committed:
resolved next steps

matifali committed:
Add icons to documentation manifest entries

Integrate visual icons into the docs manifest to enhance the
navigation experience by associating relevant symbols with
documentation sections.

stirby committed:
add steps for creating a template from an existing one in the CLI

stirby committed:
cli template create hotfix

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

matifali committed:
Update external auth and FAQs documentation

- Redirect enterprise feature link to pricing page for clarity.
- Change "enterprise license" to "premium license" for consistency.

stirby committed:
Rename "Provisioners" to "External Provisioners"

Update the documentation manifest to improve clarity by renaming the "Provisioners" section to "External Provisioners." This change aligns documentation with feature scope and user expectations.

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
README in root

matifali committed:
Rename group-sync.md to idp-sync.md for clarity

Clarify documentation to distinguish between server flag-based and
organization-based sync settings for groups and roles. Reflect the
change in filename from `group-sync.md` to `idp-sync.md` to align with
the shift from focusing solely on groups to incorporating identity
provider (IDP) synchronization more broadly.

matifali committed:
Update API and CLI documentation index filenames

Refactor the index filenames in the API and CLI documentation
generation scripts from `README.md` to `index.md` for consistency
with modern documentation standards.

bpmct committed:
Add LAN icon for offline deployment docs

Include a LAN icon in the documentation manifest to visually represent offline or air-gapped deployment sections, enhancing user navigation.

stirby committed:
fixed landing page parsing

matifali committed:
Rename cli README.md to index.md

Update the documentation structure for consistency by renaming all
`README.md` files in the CLI documentation to `index.md`. This change
aligns with modern documentation practices, improving navigation and
accessibility.

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

bpmct committed:
Add external database guide to tutorials

Integrate the "Using an External Database" guide into the tutorials section, improving accessibility. This relocation aligns with user needs to easily find setup instructions for using external databases with Coder.

matifali committed:
Simplify state arrays in docs manifest.json

Consolidate the "state" arrays for each documentation section to use inline notation, improving readability and maintaining consistency across the configuration. This change ensures a cleaner and more concise JSON structure.

matifali committed:
Fix formatting in external-auth and idp-sync docs

Improve readability by adjusting line breaks for clearer sections.

stirby committed:
returning releases

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
template dependencies

stirby committed:
template dependencies typo

matifali committed:
Rename 'Group Sync' to 'IDP Sync' in docs manifest

stirby committed:
fixed links in workspace lifecycle

matifali committed:
Update group sync references to IDP sync

Replace outdated "group-sync.md" references with "idp-sync.md" across
documentation to align with recent changes. This enhances consistency
and clarity for users following configuration and identity provider
integration instructions.

stirby committed:
user guides fixups

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
links in workspace management

matifali committed:
Correct path for security documentation link

stirby committed:
typo in jetbrains guide

stirby committed:
removed commented content from user guides

matifali committed:
Add "Open in Coder" template documentation

Include detailed instructions for embedding an "Open in Coder"
button, setting up git authentication, and modifying templates
to auto-clone repositories.

matifali committed:
Update line wrapping in Markdown docs

Improve readability by adjusting line wrapping for better alignment with style standards in 'open-in-coder.md' and 'groups-roles.md'.

matifali committed:
Capitalize "Remote Desktop" title in manifest.json

stirby committed:
broken link in api index:

bpmct committed:
Update documentation for feature stages and availability

Refine stages to better explain Alpha, Beta, and GA availability. Include guidance for feature usage and opt-in recommendations for early access and experimental stages to ensure proper understanding of development phases.

bpmct committed:
Fix broken link in security advisory table

stirby committed:
in-product links

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
premium tags appended to enterprise

stirby committed:
audited license badges and feature stages

bpmct committed:
Add markdown link check disable and enable comments

bpmct committed:
Disable markdown link check in changelog file

bpmct committed:
Remove duplicate FAQs documentation

Remove the duplicate `faqs.md` file from `docs/` as it is now covered in `docs/tutorials/faqs.md`. This ensures that the documentation remains organized without redundant files.

stirby committed:
3 bottom up redirs

bpmct committed:
Update internal documentation links

Correct and update various internal links across documentation files
to ensure consistent navigation throughout the documentation set.
This enhances usability and reduces the potential for broken links.

stirby committed:
Update links in templates documentation

Resolve incorrect relative paths across various templates documentation
files to ensure all links work seamlessly after recent refactoring.

- Corrected links to setup and workspace lifecycle docs
- Updated GitHub repository links for JFrog templates
- Fixed CLI reference paths for coder commands
- Adjusted path for template schedule image

These changes improve document navigation and maintain consistency.

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
typo

bpmct committed:
Remove markdown link check comments in changelogs

These comments are unnecessary as our links are now verified and stable.

stirby committed:
broken links

bpmct committed:
Update links in validated architectures and IDP sync docs

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

bpmct committed:
Fix image paths in IDP sync documentation

bpmct committed:
Fix broken links in IDP sync documentation

Adjust image and link paths for accuracy:
- Correct group error image path.
- Update server reference and configuration links.

stirby committed:
links

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

bpmct committed:
Fix SCIM configuration link in OIDC auth docs

stirby committed:
1/3

stirby committed:
Merge branch 'restructure-new' of https://github.com/coder/coder into restructure-new

bpmct committed:
Update paths for configuration and image links

Update the links in IDP sync documentation to align with the new file structure, ensuring correct navigation and display of images and configurations.

stirby committed:
proxies dead link

bpmct committed:
Merge branch 'main' into restructure-new

bpmct reviewed (approved):
We went through a thorough QA session and it's ready. There may be some slip-ups but we're ready to 🚢

bpmct committed:
fmt

bpmct committed:
gen

bpmct committed:
Remove deprecated docs and adjust manifest formatting

This commit eliminates outdated documentation files and updates the JSON formatting in the docs manifest for improved readability and consistency.

bpmct committed:
fmt

stirby committed:
fixed user-guides double icon

stirby committed:
corrected screenshot

matifali commented:
/cherry-pick test-cherry-pick-2.10
